ManatiCR/retopais
# RP-5: Alliances page

https://manati.atlassian.net/browse/RP-5

How-to-test:
- Create some alliances nodes
- Visit /alianzas, it should look like the prototype (https://www.estudiomanati.com/prototipos/retopais/#p=alianzas)
